src/commcare_cloud/ansible/roles/keepalived/README.md

```diff
@@ -0,0 +1,57 @@
+ansible-keepalived: Ansible role for setting up keepalived 
+============================================================
+Forked of:-https://github.com/tcomerma/ansible-keepalived/ with some changes
```

Comment:
Is this customized to our use-case, if not we can use ansible-galaxy, right?

Reply:
Yes it's customized to our use case. couple of changes in config and a new notify script.